elide-async/src/main/java/com/yahoo/elide/async/operation/JSONAPIAsyncQueryOperation.java

```diff
@@ -81,7 +84,11 @@ private String getPath(URIBuilder uri) {
     public Integer calculateRecordCount(AsyncQuery queryObj, ElideResponse response) {
         Integer count = null;
         if (response.getResponseCode() == 200) {
-            count = JsonPath.read(response.getBody(), "$.data.length()");
+            try {
+                count = JsonPath.read(response.getBody(), "$.data.length()");
+            } catch (PathNotFoundException e) {
```

Comment:
When is the path not found?

Reply:
removed the logic. It only happens during error scenarios, we will not call it in that situation,